scalalib/overrides-2.13.0/scala/Symbol.scala

```diff
@@ -0,0 +1,31 @@
+package scala
+
+// Ported from Scala.js.
+// Modified to use collection.mutable.Map instead of java.util.WeakHashMap.
+
+final class Symbol private (val name: String) extends Serializable {
```

Comment:
Why do we have different versions of this file for 2.13.x instead of just one 2.13? Why do we even have one specific to 2.13?

Reply:
I've copied Scala.js implementation. In 2.13.2 `Symbol.toString` implementation was changed. That's why we have an override for 2.13.0 and 2.13.1

Reply:
Ah right. However we don't support 2.13.{0-2}, so we can stick with the 2.13.3 version in `-2.13`. That way we'll just use the newer version.